docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -941,6 +942,22 @@ Therefore, authenticated users can access the secured application area without r
 Default resolution can also work for Bearer token authentication.
 Still, it might be less practical because a tenant identifier must always be set as the last path segment value.
 
+[[header-based-tenant-resolver]]
+==== Resolve tenants with a token header name
+
+OIDC tenants which support Bearer token authentication can be resolved using the header name of the custom HTTP header that contains a bearer token.
+The header-based resolution is enabled with the `quarkus.oidc.token.resolve-tenant-with-header` property. For example:
```

Comment:
Hi Michal, thanks, why did you choose to set it as a `quarkus.oidc.token.resolve-tenant-with-header` ? I thought we'd have something similar to `quarkus.oidc.tenant-paths`, in this case, `quarkus.oidc.tenant-header=Custom-Authorization1`